storage/buckets/main_test.go

```diff
@@ -15,41 +16,54 @@ import (
 	"golang.org/x/net/context"
 )
 
-var bucketName = fmt.Sprintf("golang-example-buckets-%d", time.Now().Unix())
+var (
+	c          *storage.Client
```

Comment:
s/c/storageClient/

Variables names should approximately scale with the scope they're used in

Reply:
Done.